components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/AuthorizationGrantCache.java

```diff
@@ -155,6 +158,9 @@ public void addToCacheByCode(AuthorizationGrantCacheKey key, AuthorizationGrantC
     public AuthorizationGrantCacheEntry getValueFromCacheByCode(AuthorizationGrantCacheKey key) {
         AuthorizationGrantCacheEntry cacheEntry = super.getValueFromCache(key);
         if (cacheEntry == null) {
+            if (log.isDebugEnabled()) {
+                log.debug("Getting cache entry from session store using authorization code");
```

Comment:
Put some context identifier to identify the log. Ex. key.getUserAttributesId(

Reply:
That AuthorizationGrantCacheKey has either authorization code/ token. That is why I didn't log them

Reply:
added two logs. if tokenis loggable, then logged the hash of the accesstoken/code. Else logged only the log string